Lesson22/todo-app/aysel/styles.css

```diff
@@ -0,0 +1,84 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    margin: 0;
+    background-color: #f8f8f8;
+    font-family: Arial, sans-serif;
+    background-image: url("https://s3.amazonaws.com/shecodesio-production/uploads/files/000/130/908/original/cHJpdmF0ZS90ZW1wbGF0ZXMvZmlsZXMvY3JlYXRlX3Rvb2wvMjAyMy0wNy8wMWdwamRubTNlOGFlemRueTNwdzVweW4ycy1sazlmdnE2ZC5wbmc.png?1719694901");
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+  
+  .container {
+    margin:50px auto;
+    max-width: 600px;
+    background-color: transparent;
+    border: 2px solid rgba(255, 255, 255, 0.2);
+    backdrop-filter: blur(20px);
+    box-shadow: 0 0 10px rgba(0 0 0 0.2);
+    padding: 20px;
+    border-radius: 15px;
+  }
+  
+  h1 {
+    text-align: center;
+    color: #333;
+  }
+  
+  form {
+    display: flex;
+    margin-bottom: 20px;
+  }
+  
+  input[type="text"] {
+    flex: 1;
+    padding: 10px;
+    font-size: 16px;
+    border: none;
+    border-bottom: 2px solid #ddd;
+  }
+  
+  button[type="submit"] button[type="reset"]{
+    padding: 12px 24px; 
+  font-size: 16px; 
+  color: white; 
+  border: none; 
+  border-radius: 5px; 
+  cursor: pointer; 
+  transition: background-color 0.3s, color 0.3s; 
+  }
+  
```

Comment:
Apart from the styles defined above for the buttons, I'd add different colors to the submit and reset buttons to make the UI more user friendly and increate the contrast between background color of button and it's text (now it's very hard to read):
```
  button[type="submit"] {
    background-color: rgb(43, 77, 70);
  }
  button[type="submit"]:hover {
    color: rgb(43, 77, 70);
    background-color: #fff;
  }
  button[type="reset"] {
    background-color: rgb(109, 41, 41);
  }
  button[type="reset"]:hover {
    color: rgb(109, 41, 41);
    background-color: #fff;
  }
```